### PR TITLE
Enhancement:  customize calendar's navigation icons

### PR DESCRIPTION
**Pull request by kellygorr**
#### Pull request checklist

- [0] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

Enhancement: added ability to customize calendar's navigation icons.

Screenshots:
screenshotarrows
screenshotarrows2

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
can you revert the package-lock.json file? There also appear to be a number of style updates as well. Can you describe all the changes coming in this PR? Or else break it up into smaller PRs.

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.Props.ts [hunk 1]
```diff
@@ -144,4 +155,29 @@ export interface ICalendarStrings {
...
+  */
+  dayPickerAriaLabel?: string;
 }
+
+export interface ICalendarIconStrings {
+  /**
+  * FabricMDL2Icons name for the left navigation icon
+  * @defaultvalue  'ChevronLeft'
+  */
+  leftNavigation: string;
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.Props.ts]: if these have defaults they don't need to be required

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss [hunk 2]
```diff
@@ -230,6 +230,9 @@ $Calendar-dayPicker-margin: 10px;
   cursor: pointer;
 }
 
+.wrap div.headerToggleView:focus {
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: is div required here? is .wrap required?

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss [hunk 3]
```diff
@@ -486,6 +489,113 @@ $Calendar-dayPicker-margin: 10px;
     }
   }
 
+  // Show the month picker over date picker (1 column layout), if enabled
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: add a screenshot of these new styles in PR

Comment (kellygorr) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: Oops, I didn't realize that there were all these .scss files, they shouldn't be here.  I will get rid of them.  They aren't apart of this PR.

Comment (kellygorr) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: It looks like updates from another pull request are also here, I'm going to sort it out, sorry!

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: I know how that goes! Done that way too many times  :)

Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: I'll close this PR and you can open one when you're ready

Comment (kellygorr) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: Thanks!  That looks like the easiest way to fix it, thank you!

**Pull request closed without merging**